Gordon360/Controllers/NewsController.cs

```diff
@@ -110,6 +111,19 @@ public ActionResult<IEnumerable<StudentNewsCategoryViewModel>> GetCategories()
             return Ok(result);
         }
 
+        [HttpGet]
+        [Route("unapproved")]
+        [StateYourBusiness(operation = Operation.UPDATE, resource = Resource.NEWS_APPROVAL)]
+        public ActionResult<IEnumerable<StudentNewsViewModel>> GetNewsUnapproved()
```

Comment:
Why is the stateyourbusiness for Operation.UPDATE and for Resource.NEWS_APPROVAL? It's a little bit misleading and probably deserves a new resource or controller logic instead.